hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/ChunkUtils.java

```diff
@@ -291,10 +301,18 @@ public static void verifyChunkFileExists(File file)
   }
 
   @VisibleForTesting
-  static <T> T processFileExclusively(Path path, Supplier<T> op) {
+  static <T> T processFileExclusively(Path path, Supplier<T> op)
+      throws InterruptedException {
+    long period = 1;
     for (;;) {
       if (LOCKS.add(path)) {
         break;
+      } else {
+        Thread.sleep(period);
+        // exponentially backoff until the sleep time is over 1 second.
+        if (period < 1000) {
+          period *= 2;
```

Comment:
Should we reset the period when the lock is acquired?

Reply:
I don't think it's necessary.  `period` is not used after the lock is acquired.  Other callers will start from 1, since it's a local variable.

Reply:
Got it, thanks for the clarification.